### PR TITLE
[CHI-393] Fix Badge CE size inconsistencies

### PR DESCRIPTION
**Pull request by jllr**
@janibolkvadze 

https://ctl.atlassian.net/browse/CHI-393

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/360/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/361/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/362/ for more details.

File: src/custom-elements/src/components/badge/badge.tsx [hunk 1]
```diff
@@ -31,14 +31,14 @@ export class Badge {
...
-   *  to set size of a badge { small or smaller }.
+   *  to set size of a badge { xs or sm }.
    */
   @Prop({ reflectToAttr: true }) size: string;
 
   @Watch('size')
   sizeValidation(newValue: string) {
-    if (newValue && !['', 'small', 'smaller'].includes(newValue)) {
-      throw new Error(`${newValue} is not a valid size for badge. Valid values are small, smaller or ''. `);
+    if (newValue && !['', 'small', 'smaller', , 'sm', 'xs'].includes(newValue)) {
```
Comment (janibolkvadze) [inline src/custom-elements/src/components/badge/badge.tsx]:
```suggestion
    if (newValue && !['', 'small', 'smaller', 'sm', 'xs'].includes(newValue)) {
```

Please remove this comma

Comment (janibolkvadze) [inline src/custom-elements/src/components/badge/badge.tsx]:
```suggestion
    if (newValue && !['', 'small', 'smaller', 'sm', 'xs'].includes(newValue)) {
```
Please remove the unnecessary comma

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/364/ for more details.

**Pull request merged**